test/server/graphql/v2/mutation/CreateCollectiveMutations.test.js

```diff
@@ -0,0 +1,44 @@
+import { expect } from 'chai';
+
+import * as utils from '../../../../utils';
+import models from '../../../../../server/models';
+
+const createCollectiveQuery = `
+    mutation createCollective($collective: CollectiveCreate!) {
+      createCollective(collective: $collective) {
+        name
+        slug
+        tags
+      }
+    }`;
+
+const newCollectiveData = {
```

Comment:
The test is helpful to show an example of the kind of data I will need to input. Thanks! 🙌